example/lib/main.dart

```diff
@@ -104,18 +98,18 @@ class _MyHomePageState extends State<MyHomePage> {
             ),
             GanttChartView(
               maxDuration: const Duration(days: 30 * 2),
-              startDate: DateTime(2022, 6, 7),
+              startDate: DateTime.now(),
               dayWidth: dayWidth,
               eventHeight: 40,
               stickyAreaWidth: 200,
               showStickyArea: showStickyArea,
               showDays: showDaysRow,
-              weekEnds: const {WeekDay.friday, WeekDay.saturday},
+              weekEnds: const {WeekDay.saturday, WeekDay.sunday},
```

Comment:
please revert `weekEnds` and `startOfTheWeek` back, since in this example I am showing the default values

Reply:
Super interesting, I suspected that Egypt only did week-ends on Friday Saturday,
Most of the world do Saturday Sunday, or Sunday only. 
It's cool to keep Egyptian vibes and through this to curb ethnocentrism a bit.

![wekeendmap-eng](https://user-images.githubusercontent.com/98157423/211559996-8ae3a12b-7478-4180-8528-bc158ad50fca.png)

Reply:
well, it's ok if you want to change the default weekend values, what I meant is that you need to also change the readme as well

my way of thinking is that if a user can change the behavior anyway, the defaults are irrelevant